test/Solana.SvmCpiEventsClient.Integration.test.ts

```diff
@@ -62,6 +75,83 @@ describe("SvmCpiEventsClient (integration)", () => {
     const signature = await deposit(signer, solanaClient, depositInput, decimals);
     return { signature, depositInput };
   };
+  const uint8ArrayToBigNumber = (arr: Uint8Array): BigNumber => {
+    if (arr.length === 0) {
+      return BigNumber.from(0); // or handle this case appropriately
+    }
+
+    const hex = Array.from(arr)
+      .map((byte) => byte.toString(16).padStart(2, "0"))
+      .join("");
+
+    return BigNumber.from("0x" + hex);
+  };
```

Comment:
This should fwiw be unnecessary - BigNumber seems to handle Uint8Arrays natively.

```
> arr = new Uint8Array([255])
Uint8Array(1) [ 255 ]
> ethers.BigNumber.from(arr).toNumber()
255

```